app/views/steps/_form.html.erb

```diff
@@ -59,11 +59,67 @@
 </div>
 
 <div class="form-group">
-  <div>
-    <p>
-      <%= step.broken_links %>
-    </p>
-  </div>
+  <% if step.broken_links && step.broken_links.count > 0 %>
+    <div class="broken-links">
+      <div class="panel-group panel-warning" id="accordion" role="tablist" aria-multiselectable="true">
+        <div class="panel-heading">
+          <%= render 'shared/steps/broken-links-summary', step: step %>
+        </div>
+        <% step.broken_links.each.with_index do |link, index| %>
+          <div class="panel panel-warning">
+            <div class="panel-heading" role="tab" id="brokenLinkHeading<%= index %>">
+              <h4 class="panel-title">
```

Comment:
The heading immediately preceding these H4s is a H1. I'm not sure it needs to be a heading - you could just use the `.h4` class if the font size is what you were after.

Reply:
This was the default bootstrap code - I could modify it and put a `p` tag instead